lib/authc/OAuthBasicExchangeAuthenticator.js

```diff
@@ -76,7 +120,7 @@ OAuthBasicExchangeAuthenticator.prototype.buildAccesstoken = function buildAcces
     exp: now + self.ttl
   };
 
-  var scope = self.scopeFactory(account,self.requestedScope);
+  scope = self.scopeFactory(account, self.requestedScope);
```

Comment:
Bug: This line should be removed; `scope` is sent into `buildAccessToken` from line `104` with the `optionalResult` from line `87`.